x-pack/docs/en/security/authentication/saml-guide.asciidoc

```diff
@@ -853,3 +853,171 @@ A conscious effort has been made to mask this complexity with sane defaults and
+defined as a URL which the custom web application handles. When it receives this HTTP POST request, the
+custom web application must parse it and make an HTTP POST request itself to the
+`_security/saml/authenticate` API. It must authenticate as the `saml-service-user` user and pass
+the Base64 encoded SAML Response that was sent as the body of the request. It must also pass the value for `id` that it had saved in the user's session previously.
++
+See {ref}/security-api-saml-authenticate.html[SAML authenticate API] for more details.
++
+[source,js]
+-----------------------------------------------------------------------
+POST /_security/saml/authenticate
+{
+  "content" : "PHNhbWxwOlJlc3BvbnNlIHhtbG5zOnNhbWxwPSJ1cm46b2FzaXM6bmFtZXM6dGM6U0FNTDoyLjA6cHJvdG9jb2wiIHhtbG5zOnNhbWw9InVybjpvYXNpczpuYW1lczp0YzpTQU1MOjIuMD.....",
+  "ids" : ["4fee3b046395c4e751011e97f8900b5273d56685"]
+}
+-----------------------------------------------------------------------
+// CONSOLE
+// TEST[skip:handled in IT]
++
+Elasticsearch will validate this and if all is correct will respond with an access token that can be used
+as a `Bearer` token for subsequent requests. It also supplies a refresh token that can be later used to refresh the given
+access token as described in {ref}/security-api-get-token.html[get token API].
+. The response to calling `/_security/saml/authenticate` will contain only the username of the authenticated
+user. If you need to get the values for the SAML Attributes that were contained in the SAML
+Response for that user, you can call the Authenticate API `/_security/_authenticate/` using the access token as a `Bearer` token
+and the SAML attribute values will be contained in the response as part of the <<saml-user-metadata>>.
+
+
+==== Handling the IdP-initiated authentication flow
+
+{es} can also handle the IdP-initiated Single Sign On flow of the SAML 2 Web Browser SSO profile. In this
+case the authentication starts with an unsolicited authentication response from the SAML Identity
+Provider. The difference with the <<saml-sp-init-sso, SP initiated SSO>> is that the web application needs to handle
+requests to the `sp.acs` that will not come as responses to previous redirections. As such, it will not have a session
+for the user already, and it will not have any stored values for the `id` parameter. The request to the
+`_security/saml/authenticate` API will look like the one below in this case:
+
+
+[source,js]
+-----------------------------------------------------------------------
+POST /_security/saml/authenticate
+{
+  "content" : "PHNhbWxwOlJlc3BvbnNlIHhtbG5zOnNhbWxwPSJ1cm46b2FzaXM6bmFtZXM6dGM6U0FNTDoyLjA6cHJvdG9jb2wiIHhtbG5zOnNhbWw9InVybjpvYXNpczpuYW1lczp0YzpTQU1MOjIuMD.....",
+  "ids" : []
+}
+-----------------------------------------------------------------------
+// CONSOLE
+// TEST[skip:handled in IT]
+
+==== Handling the logout flow
```

Comment:
This section URL defaults to "_handling_the_logout_flow", I suggest something like this:
```suggestion
[[saml-logout]]
==== Handling the logout flow
```